### ISSUE TITLE
[🐛 Bug]: Accessibility Selector has trouble to find input by for a11y name

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/webdriverio#8826]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

latest

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
n/a
```

### What happened?

Given a form like:

```html
      <form aria-label="Payment form" data-v-3b3d1f66="">
        <div>
          <label for="name">Name</label>
          <input type="text" name="name" id="name" autocomplete="off" required="" />
        </div>
        <div data-v-3b3d1f66="">
          <button id="submit-payment" type="submit" data-v-3b3d1f66="">Submit</button>
        </div>
      </form>
```

If we try to find the name input by the accessibility name we get back 3 results from which the input is listed last and the `div` first causing WebDriver to fail due to `not interactable`.

![xpath](https://user-images.githubusercontent.com/731337/188518952-0228c927-8001-4c77-b571-41df3880dd77.gif)

### What is your expected behavior?

I expect that the `input` element would be found rather than the `div`. I am not sure why the order is like that given that the input xpath is first in the union expression.

### How to reproduce the bug.

Go to http://xpather.com/ and enter the html code from above and the following xPath

```
(
  .//*[@aria-labelledby=(//*[normalize-space() = "Name"]/@id)] |
  .//*[@aria-describedby=(//*[normalize-space() = "Name"]/@id)] |
  .//*[@aria-label = "Name"] |
  .//input[@id = (//label[normalize-space() = "Name"]/@for)] |
  .//input[@placeholder="Name"] |
  .//input[@aria-placeholder="Name"] |
  .//*[@title="Name"] | .//img[@alt="Name"] |
  .//*[normalize-space() = "Name"]
)
```

### Relevant log output

```typescript
n/a
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by jayrepo** [comment]
Browsers now only support XPath 1.0, according to the [spec](https://www.w3.org/TR/1999/REC-xpath-19991116/), the return value is an unordered collection of nodes.
In Chrome's implementation, the nodes are in document order.

workarounds might be:
 - Send multiple requests and combine the results in pre-defined order
 - Send single request by executing JavaScript to evaluate multiple XPaths expressions.

**Comment by christian-bromann** [comment]
Fixed by https://github.com/webdriverio/webdriverio/pull/11586
